apis/autoscaling/v1alpha1/hvpa_types.go

```diff
@@ -349,3 +350,6 @@ type HvpaList struct {
 /*func init() {
 	SchemeBuilder.Register(&Hvpa{}, &HvpaList{})
 }*/
+
+// Hub marks this version as the API conversion Hub
+func (*Hvpa) Hub() {}
```

Comment:
Just a random thought: In the hypothetical long run, the v1alpha2 may turn out to be the more natural hub, since  it's likely to be closer to a hypothetical future version.

Not something I'd change at this point though. Besides, iiuc, it has no practical significance, since it's just a matter of moving that one line of code, which can be done effortlessly at any point in time.